test/helpers/fixture_sets/basic.rb

```diff
@@ -0,0 +1,63 @@
+module FixtureSetAssertions
+  class Basic < FixtureSet
```

Comment:
I don't have a strong opinion on leaving the helpers here or in the integration tests. It certainly helps break up things later, I think what that comes down to is how many fixture sets do we expect to have? Do you expect 2? 5? 10? I think that would also impact some of the other work in this PR if we don't really expect to ever need more than `HelloCloud`. I think you definitely made the right call to make it easy to manipulate the set instead of creating a lot of new ones, but perhaps people will find that so convenient they won't create sets altogether. Do you see any use-case for having more than one fixture set?